### DISCUSSION TITLE
ImportError: cannot import name 'asmatrix' from 'scipy.sparse.sputils'

### DISCUSSION THREAD

**Discussion by Natibliss** [lllyasviel/Fooocus#3303] [Q&A] [answered]
First of all I want to thank the developers, they do an excellent job, thank you very much, my respects.

anyone with the same error?

I use Fooocus on Kaggle, yesterday I used it perfectly but today I can't use it.

File "/opt/conda/lib/python3.10/site-packages/scipy/sparse/linalg/isolve/utils.py", line 7, in <module>
    from scipy.sparse.sputils import asmatrix
ImportError: cannot import name 'asmatrix' from 'scipy.sparse.sputils' (/opt/conda/lib/python3.10/site-packages/scipy/sparse/sputils.py)
connect_to localhost port 7865: failed.
connect_to localhost port 7865: failed.

**Comment by mashb1t** [comment, marked as answer]
see https://github.com/lllyasviel/Fooocus/discussions/3284#discussioncomment-10078471

**Reply by Penny13692018** [reply]
Have the same error, but I tried in Kaggle notebook with 
>. HOME_FOLDER = '/kaggle/working/Fooocus/'
%cd $HOME_FOLDER
!pip install -r requirements_versions.txt

Still not working

**Comment by suxnet** [comment]
ChatGPT says we gotta downgrade scipy. Guess what? It works!

pip install scipy==1.8.0

**Reply by suxnet** [reply]
PS: downgrade scipy BEFORE running the Pinggy cell
